app/templates/eventView.html

```diff
@@ -1,40 +1,23 @@
 {% set title = eventData.name %}
-{% extends "base.html" %}
+
+{% set eventName = eventData.name %}
+{% set eventProgram = eventData.program %}
+{% set eventProgramName = eventData["program"].programName %}
+{% set eventPast = event.isPast %}
+{% set eventID = eventData.id %}
```

Comment:
Let's clean up the data we are sending in, since we are having to specify it multiple times. There are enough different values here that we should consider just using eventData and eventPast in events_nav.html